### PR TITLE
Fix Java Comment RegEx Bug

### PR DESCRIPTION
**Pull request by dchen97**
* Fixed bug where the ZeroDivisionError would get blocked by the Java Comment Regex for Wrong Language Syntax Detection

### CODE REVIEW

File: client/question/question.js [hunk 1]
```diff
@@ -248,7 +248,7 @@ tie.constant('WRONG_LANGUAGE_ERRORS', {
   }, {
     // Used the Java comment syntax
     errorName: 'javaComment',
-    regExString: '\\/(\\s*|\\w*)*\\n|\\/\\*(\\*)?(\\s*|\\w*)*\\*\\/',
+    regExString: '\\/\\/(\\s*|\\w*)*\\n|\\/\\*(\\*)?(\\s*|\\w*)*\\*\\/',
```
Comment (seanlip) [inline client/question/question.js]: Btw, are the two inner asterisks in `(\\s*|\\w*)*` (both occurrences) necessary? (Wouldn't the asterisk outside the parens take care of that?)

**Comment by dchen97** [comment]
I think in the confusion of all the double backslashes I just forgot one ><

File: client/question/question.js (hunk 1 above)
Comment (dchen97) [inline client/question/question.js]: No, they don't appear to be necessary. I'll delete them.

Comment (dchen97) [inline client/question/question.js]: Done!

**Pull request merged**